staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -206,14 +222,28 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 		return false, webhookerrors.ToStatusErr(h.Name, result.Result)
 	}
 
+	patchAnnotationKey := PatchAuditAnnotationPrefix + h.Name
```

Comment:
Name is not globally unique. We need to qualify it at least with the webhook configuration name, e.g.:

https://github.com/kubernetes/kubernetes/blob/661e4d5b3857b5268f2d6d6b79219daf2e4b5833/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go#L27

Reply:
good point. Added webhook config name/UID into consideration